x-pack/plugins/index_management/__jest__/client_integration/helpers/template_form.helpers.ts

```diff
@@ -135,6 +137,29 @@ export const formSetup = async (initTestBed: SetupFunc<TestSubjects>) => {
     }
   };
 
+  const clickReviewStep = async ({ name, indexPatterns }) => {
+    const { form, find, component, waitFor } = testBed;
+
+    if (name) {
+      form.setInputValue('nameField.input', name);
+    }
+
+    if (indexPatterns) {
+      const indexPatternsFormatted = indexPatterns.map((pattern: string) => ({
+        label: pattern,
+        value: pattern,
+      }));
+
+      find('mockComboBox').simulate('change', indexPatternsFormatted); // Using mocked EuiComboBox
+      await nextTick();
```

Comment:
Same here, don't use `nextTick()` we will have to remove all of them in other tests.